README.md

```diff
@@ -1,15 +1,41 @@
 
 <div align="center">
 
-# DBOS Transact: A Lightweight Durable Execution Library Built on Postgres
+# DBOS Transact: Lightweight Durable Python Workflows
 
 #### [Documentation](https://docs.dbos.dev/) &nbsp;&nbsp;•&nbsp;&nbsp;  [Examples](https://docs.dbos.dev/examples) &nbsp;&nbsp;•&nbsp;&nbsp; [Github](https://github.com/dbos-inc) &nbsp;&nbsp;•&nbsp;&nbsp; [Discord](https://discord.com/invite/jsmC6pXGgX)
 </div>
 
 ---
 
-DBOS Transact is a Python library for **ultra-lightweight durable execution**.
-For example:
+## What is DBOS?
+
+DBOS provides lightweight durable workflows built on top of Postgres.
+Instead of managing your own workflow orchestrator or task queue system, you can use DBOS to add durable workflows and queues to your program in just a few lines of code.
+
+To get started, follow the [quickstart](https://docs.dbos.dev/quickstart) to install this open-source library and connect it to a Postgres database.
+Then, annotate workflows and steps in your program to make it durable!
+That's all you need to do&mdash;DBOS is entirely contained in this open-source library, there's no required infrastructure for you to configure or manage.
+
+## When Should I Use DBOS?
+
+You should consider using DBOS if you're **concerned about how your application handles failures**.
```

Comment:
I think "handles failures" is not accurate enough. We can focus on reliability + checkpointing for long-running tasks.

```suggestion
You should consider using DBOS if **you care about reliably making progress**.
```